### PR TITLE
Add travis CI

### PR DESCRIPTION
**Pull request by feiskyer**
This PR adds initial verify scripts and travis CI.

nit: It will fail because there is no go files.

### CODE REVIEW

**Comment by xlgao-zju** [comment]
Do we need `golint` in CI?

**Comment by feiskyer** [comment]
> Do we need golint in CI?

Of course. Added in the latest commit.

**Comment by xlgao-zju** [comment]
Cool

File: hack/verify-all.sh [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SILENT=true
+KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
```
Comment (xlgao-zju) [inline hack/verify-all.sh]: Maybe `CRI_TOOLS_ROOT` rather than `KUBE_ROOT`?

**Comment by feiskyer** [comment]
@Random-Liu @yujuhong Travis enabled. PTAL.

**Comment by mrunalp** [comment]
LGTM

**Comment by xlgao-zju** [comment]
Ping @Random-Liu

**Comment by ideahitme** [comment]
@here please consider using https://github.com/kubernetes/repo-infra/tree/master/verify to avoid code duplication

**Comment by feiskyer** [comment]
@ideahitme Thanks. Updated to repo-infra.

**Comment by ideahitme** [comment]
@feiskyer normally you would do it via vendoring or git-submodule, would be easier for you to upgrade and manage it in the future. Take a look at https://github.com/kubernetes-incubator/external-dns/ for reference

**Comment by feiskyer** [comment]
@ideahitme Yep, vendor is more easier to upgrade but godeps doesn't support manage none-golang packages. So this PR just get it via `git subtree`.

**Pull request merged**